bin/build_code_changes_32.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+mode="side-by-side"
+basedir=`cd $(dirname "$BASH_SOURCE"); cd ..; pwd`
+outdir=`cd "$basedir"; cd ./web/code-changes/; pwd`
+datadir=`cd "$basedir"; cd ./data/code-changes/; pwd`
+latest="3.2.1"
+releases="3.2.0"
+
+# Set up the file structure
+if [ ! -d "$datadir/repo" ];
+then
+	mkdir "$datadir"/repo
+fi
+
+if [ ! -d "$datadir/versions" ];
+then
+	mkdir "$datadir"/versions
+fi
+
+# Clone the repository
+if [ ! -d "$datadir/repo/phpBB" ];
+then
+	git clone git://github.com/phpbb/phpbb3.git "$datadir"/repo
```

Comment:
It just a copy/paste of the other script. We really need to cleanup and few things in this repo